### ISSUE TITLE
Bind Snapclient / Server to a specific Network Interface?

### ISSUE THREAD

**Issue by almostserious** [badaix/snapcast#739]
On my PI I have both WiFi and LAN active. I would like to specify, that the client only binds itself to a specific network interface.
i.e.: Just restarted, and the snapclient was running on the WiFi Interface (different MACID). 
Is it already possible to add to the config to only connect via the Ethernet Interface?

**Comment by badaix** [comment]
No, it's not possible, only the server can bind to specific interfaces.
Pull requests are welcome :)
